activerecord/test/cases/adapters/postgresql/schema_test.rb

```diff
@@ -530,6 +531,15 @@ def test_non_default_opclass_is_dumped
 
     assert_match(/opclass: \{ description: :text_pattern_ops \}/, output)
   end
+
+  def test_wrapped_column_when_creating_gin_index
+    @connection.enable_extension("pg_trgm")
+    @connection.execute "CREATE INDEX trains_position ON trains USING gin(position gin_trgm_ops)"
```

Comment:
Can you add an extra test case that mixed with default opclass like `@connection.execute "CREATE INDEX trains_name_and_description ON trains USING btree(name, position text_pattern_ops)"`?